spinn_utilities/timer.py

```diff
@@ -12,8 +12,42 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import sys
 
-import datetime
+
+if sys.version_info >= (3, 7):
+    # acquire the most accurate measurement available (monotonic_ns)
+    from time import monotonic_ns as now # pylint: disable=no-name-in-module
+
+    # have to convert to a timedelta for rest of code to read
+    from future.backports.datetime import timedelta # pylint: disable=no-name-in-module
```

Comment:
Doesn't datetime include timedelta in 3.7?

Reply:
same issue. given the result from the montonic is a float. had to improt datetime which at python 3 i assumed we need to go back

Reply:
https://docs.python.org/3/library/datetime.html

Again, it is in 3.7 I am fairly sure...